Assets/Scripts/MenuButton.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuButton : MonoBehaviour
+{
+	[SerializeField] MenuButtonController menuButtonController;
+	[SerializeField] Animator animator;
+	[SerializeField] AnimatorFunctions animatorFunctions;
+	[SerializeField] int thisIndex;
+	public bool isSceneChanger;
```

Comment:
don't use public. it brings security issues and it will be ugly to access something using lower case variables